docs/contributor/adr/0007-rate-limit.md

```diff
@@ -0,0 +1,28 @@
+# Rate Limit
+
+## Status
+
+Draft
+
+## Context
+
+It has been decided to indroduce rate limit functionality into Kyma platform to allow users to set rate limit functionality on the service mesh layer, therefore allowing to consume intended service mesh functionality which is abstracting away networking concerns outside applications inside the mesh.
+Since Istio is an underlying service mesh responsible for the workloads networking across Kyma clusters, therefore it is the Istio which allows to configure such functionality.
+Taking that into account Goat Team is responsible to introduce this feature.
+At this moment it is not possible to introduce global rate limit provided by Istio service mesh because of technical or legal limitations related to the deployment of key-value stores in the cluster.
+
+## Decision
+
+Taking the context part into account, it was decided to implement local rate limit as the only rate limiting posibility for now.
```

Comment:
Does it make sense to explain here what does the local mean in this context (implicitly opposed to 'global')?

Reply:
Sure, it's a good idea since it can be confusing sometimes. I'll try to add some short explanation.